src/development/LocalTest/appsettings.json

```diff
@@ -36,8 +36,8 @@
   "AllowedHosts": "*",
   "LocalPlatformSettings": {
     "LocalTestingStorageBasePath": "c:/AltinnPlatformLocal/",
-    "AppRepositoryBasePath": "<Path to your repos folder where you have cloned the Altinn Studio application repositories>",
-    "LocalTestingStaticTestDataPath": "<Path to altinn-studio repo>/src/development/TestData/"
```

Comment:
This should probably not be checked in.

Reply:
Agree, at least not while this will not work on linux/mac

Reply:
yes., should not be part of the pull request. This is not the first time this file is included.